README.md

```diff
@@ -1,9 +1,102 @@
-Libero search
-=============
+# Libero Search
+The purpose of this project is to provide the ability to search indexed data via the
+`/search` endpoint.
 
-Getting help
-------------
+## Getting help
 
 - Report a bug or request a feature on [GitHub](https://github.com/libero/libero/issues/new/choose).
 - Ask a question on the [Libero Community Slack](https://libero.pub/join-slack).
 - Read the [code of conduct](https://libero.pub/code-of-conduct).
+
+
+## Dependencies
+
+* [Docker](https://www.docker.com/)
```

Comment:
And Make?

Reply:
Make is installed everywhere by default. Technically not required but makes running the software locally easier.

Reply:
(Cavaet: My knowledge of Make is pretty much just from [Symfony starting to adopt it then stopping](http://fabien.potencier.org/symfony4-flex-update.html).)

Not everywhere; Windows didn't but maybe Windows 10 fixed it with its Linux compatibility? There are different flavours of Make too.

Reply:
I could mention that in order to use the commands below `gnu make` is required. Would that suffice?

Reply:
I've updated the README to include information about `gnu make`. Please check.